Sources/Private/Utility/Extensions/AnimationKeypathExtension.swift

```diff
@@ -138,6 +144,30 @@ extension KeypathSearchable {
 
     return allKeypaths
   }
+
+  /// Computes the list of animation keypaths that descend from this layer
+  func layerKeypaths(for keyPath: AnimationKeypath? = nil) -> [CALayer: AnimationKeypath] {
```

Comment:
Adapted from the `allKeypaths` method above, but just different enough that it didn't seam feasible to consolidate the logic into a single helper